python_modules/libraries/dagster-gcp/dagster_gcp/gcs/compute_log_manager.py

```diff
@@ -60,10 +60,10 @@ def __init__(
             self._bucket = (
                 storage.Client()
                 .from_service_account_info(credentials_info)
-                .get_bucket(self._bucket_name)
+                .bucket(self._bucket_name)
```

Comment:
If the bucket doesn't exist, won't this defer throwing an error until we try to write logs (instead of when we initialize)?

If so, I think we might still want to use `get_bucket` here and elsewhere. If a step has potentially expensive computation and a bucket doesn't exist, I think I'd rather see it fail before any of that computation runs than afterward when it can't put the object.

Reply:
Good catch @jmsanders 

I've added an extra check during the initialization of the log manager., this should catch the error. I've seen it here: https://github.com/dagster-io/dagster/blob/f429612395248757a12e9e07ef94929b98ce5a55/python_modules/libraries/dagster-gcp/dagster_gcp/gcs/io_manager.py#L16-L17

Reply:
Great - I think that's a good pattern to give us the ability to "fail fast" without needing to pull all the metadata with subsequent uses.

I'll kick off the test suite and merge if things look good. Thanks!